spill/board.c

```diff
@@ -17,94 +17,98 @@
 
 #include "snake.h"
 #include "board.h"
+#include "constants.h"
 #include <stddef.h>
 #include <stdlib.h>
 #include <string.h>
 #include <stdio.h>
+#include <stdint.h>
 
-int board[BOARD_HEIGHT][BOARD_WIDTH];
-
+long int board[BOARD_HEIGHT][BOARD_WIDTH];
+int count_food;
 
 void init_board(void) {
 	// Initialize the board with only 0s (clear blocks)
 	for (int i = 0; i < BOARD_HEIGHT; i++) {
 		for (int j = 0; j < BOARD_WIDTH; j++) {
-			board[i][j] = 0;
+			board[i][j] = BLOCK_BLANK;
 		}
 	}
 }
 
-int get_square_value(int x, int y) {
+long int get_square_value(int x, int y) {
     return board[y][x];
 }
 
-void set_square_value(int x, int y, int value) {
+void set_square_value(int x, int y, long int value) {
     board[y][x] = value;
 }
 
 // To be used under initialization, put a whole snake on the board
 void add_snake_to_board(struct Body *snake_head){
```

Comment:
while `Type *variable` is part of a lot of standard  syntax guides. I personally dislike it. `Type* variable` makes more sense for defining the type of a variable like this, and it also makes it easier to see the difference between  dereferencing and typing.

I.e `*variable` means dereferencing, and `Type* variable` means typing

EDIT: In other parts of the project other things do follow the way I like it it seems